helpers/compile/plugins/fill-plugin/fillers/weakref.ts

```diff
@@ -0,0 +1,11 @@
+export class WeakRef<T> {
+  value: T
+
+  constructor(value: T) {
+    this.value = value
+  }
+
+  deref() {
+    return this.value
+  }
+}
```

Comment:
This does not exist on cloudflare workers, it cannot be polyfilled very well, we just make it do nothing.

Reply:
We might want to introduce explicit `.destroy` method to the engine later then, if we can not use WeakRefs universally.